inc/mailcollector.class.php

```diff
@@ -877,32 +883,6 @@ function buildTicket($uid, $options = []) {
          $tkt['tickets_id'] = intval($match[1]);
       }
 
-      // Double encoding for > and < char to avoid misinterpretations
-      $tkt['content'] = str_replace(['&lt;', '&gt;'], ['&amp;lt;', '&amp;gt;'], $tkt['content']);
-
-      $is_html = false;
-      //If files are present and content is html
-      if (isset($this->files)
-          && count($this->files)
-          && ($tkt['content'] != strip_tags($tkt['content']))
```

Comment:
I changed it to use the `$this->body_is_html` as a mail containing `From: <someone@mailbox.com>` was considered as HTML.